### ISSUE TITLE
Can't use the 10.2.0 version when pointing to terraform registry module

### ISSUE THREAD

**Issue by freyman-balaguera-kns** [terraform-google-modules/terraform-google-lb-http#428]
### TL;DR

I want to use the 10.2.0 version in my terraform module. But it says "newest version is 10.1.0".

### Expected behavior

Use 10.2.0 version when required in my terraform lb module 

### Observed behavior

Error: Unresolvable module version constraint
│ 
│ There is no available version of module
│ "registry.terraform.io/terraform-google-modules/lb-http/google"
│ (main.tf:17) which matches the given version constraint. The newest
│ available version is 10.1.0.

<img width="914" alt="Screenshot 2024-05-24 at 2 56 31 PM" src="https://github.com/terraform-google-modules/terraform-google-lb-http/assets/107630104/60e09a92-19cd-4912-ba9d-0758352afa09">

### Terraform Configuration

```hcl
module "gce-lb-https" {
  source  = "terraform-google-modules/lb-http/google"
  version = "~> 10.2.0"
  
  ...

}
```

### Terraform Version

```sh
1.3.10
```

**Comment by imrannayer** [comment]
@g-awmalik seems like registry is not getting updates.

**Comment by imrannayer** [comment]
fixed
